config/test_healthcare_facilities.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Since this file is only used by the test suite it would be more appropriate to put it in the test/fixtures folder.